schemas/com.snowplowanalytics.iglu/igluctl_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,201 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for Igluctl's configuration",
+	"self": {
+		"vendor": "com.snowplowanalytics.iglu",
+		"name": "igluctl_config",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"env": {
+			"description": "Target environment of the configuration",
+			"type": "string",
+			"enum": ["test", "mini", "prod"]
+		},
+		"input": {
+			"description": "Path to input schemas",
+			"type": "string",
+			"format": "uri"
+		},
+		"lint": {
+			"description": "Schema for lint command",
+			"type": "object",
+			"properties": {
+				"skipWarnings": {
+					"description": "Don't output messages with log level less than ERROR",
+					"type": "boolean",
+					"default": false
+				},
+				"includedChecks": {
+					"description": "Lint using provided linters",
+					"type": ["array", "null"],
+					"items" : {
+						"type" : "string",
+						"enum": [
+							"rootObject",
+							"unknownFormats",
+							"numericMinMax",
+							"stringLength",
+							"optionalNull",
+							"description"
+						]
+					}
+				}	
+			},
+			"required": ["includedChecks"]
+		},
+		"generate": {
```

Comment:
`generate` is missing `required`. In fact, we want to make all fields required, but possibly `null`. This approach is more templating-friendly.